processor/src/tasks/writeKitspaceYaml.ts

```diff
@@ -1,59 +1,59 @@
-import escape from 'escape-html'
-import LinkifyIt from 'linkify-it'
 import path from 'node:path'
 
+import { unified } from 'unified'
+import rehypeHighlight from 'rehype-highlight'
+import rehypeSanitize from 'rehype-sanitize'
+import rehypeStringify from 'rehype-stringify'
+import remarkEmoji from 'remark-emoji'
+import remarkGfm from 'remark-gfm'
+import remarkParse from 'remark-parse'
+import remarkRehype from 'remark-rehype'
+
 import { JobData } from '../jobData.js'
 import * as utils from '../utils.js'
 
-const linkify = new LinkifyIt()
-
-export default function writeKitspaceYaml(
+export default async function writeKitspaceYaml(
   job,
   { kitspaceYaml, outputDir }: Partial<JobData>,
 ) {
   const kitspaceYamlJson = path.join(outputDir, 'kitspace-yaml.json')
   job.updateProgress({ status: 'in_progress', file: kitspaceYamlJson })
-  const KitspaceYamlJsonLinkified = linkifyKitspaceYaml(kitspaceYaml)
+
+  const rendered = await renderKitspaceYamlSummaries(kitspaceYaml)
 
   return utils
-    .writeFile(kitspaceYamlJson, JSON.stringify(KitspaceYamlJsonLinkified, null, 2))
+    .writeFile(kitspaceYamlJson, JSON.stringify(rendered, null, 2))
     .then(() => job.updateProgress({ status: 'done', file: kitspaceYamlJson }))
     .catch(error =>
       job.updateProgress({ status: 'failed', file: kitspaceYamlJson, error }),
     )
 }
 
-function linkifyKitspaceYaml(kitspaceYaml) {
+async function renderKitspaceYamlSummaries(kitspaceYaml) {
   if (kitspaceYaml.multi) {
-    const linkifiedKitspaceYaml = kitspaceYaml
-    Object.keys(kitspaceYaml.multi).forEach(subProject => {
-      linkifiedKitspaceYaml.multi[subProject] = linkifyProjectSummary(kitspaceYaml.multi[subProject])
-    })
-    return linkifiedKitspaceYaml
-  }
-
-  return linkifyProjectSummary(kitspaceYaml)
-}
-
-function linkifyProjectSummary(kitspaceYaml) {
-  let escapedSummary = escape(kitspaceYaml.summary || '')
-  const matches = linkify.match(escapedSummary)
-
-  if (matches) {
-    for (const match of matches) {
-      // Use https by default
-      const url = new URL(match.url)
-      if (!match.schema) {
-        url.protocol = 'https:'
+    const rendered = { multi: {} }
+    for (const key of kitspaceYaml.multi) {
+      const subProject = kitspaceYaml.multi[key]
+      rendered.multi[subProject] = {
+        ...subProject,
+        summary: await renderSummary(subProject.summary),
       }
-
-      escapedSummary = escapedSummary.replace(
-        match.raw,
-        `<a href="${url.toString()}" rel="noopener noreferrer" target="_blank">${match.text}</a>`
-      )
     }
+    return rendered
   }
+  return { ...kitspaceYaml, summary: await renderSummary(kitspaceYaml.summary) }
+}
 
-  kitspaceYaml.summary = escapedSummary
-  return kitspaceYaml
+const Remarker = unified()
+  .use(remarkParse)
+  .use(remarkEmoji)
+  .use(remarkGfm)
+  .use(remarkRehype)
+  .use(rehypeHighlight)
+  .use(rehypeSanitize)
+  .use(rehypeStringify)
```

Comment:
Do we need all of this processing. Full markdown support?

Reply:
I guess we can get rid of `remarkGfm`.

Reply:
```suggestion
  .use(remarkParse)
  .use(remarkEmoji)
  .use(remarkRehype)
  .use(rehypeSanitize)
  .use(rehypeStringify)
```